pkg/apis/core/validation/validation.go

```diff
@@ -2609,6 +2610,42 @@ func validatePullPolicy(policy core.PullPolicy, fldPath *field.Path) field.Error
 	return allErrors
 }
 
+var supportedResizeResources = sets.NewString(string(core.ResourceCPU), string(core.ResourceMemory))
+var supportedResizePolicies = sets.NewString(string(core.NoRestart), string(core.RestartContainer))
+
+func validateResizePolicy(policyList []core.ResizePolicy, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) && len(policyList) > 0 {
+		allErrors = append(allErrors, field.Invalid(fldPath, "resizePolicy", "InPlacePodVerticalScaling feature is disabled"))
+		return allErrors
+	}
+
+	// validate that resource name is not repeated, supported resource names and policy values are specified
+	resources := make(map[core.ResourceName]bool)
+	for _, p := range policyList {
+		if _, found := resources[p.ResourceName]; found {
+			allErrors = append(allErrors, field.Invalid(fldPath, p.ResourceName, "resource must not be repeated"))
```

Comment:
field.Duplicate()

Reply:
fixed in commit https://github.com/vinaykul/kubernetes/pull/1/commits/5126b9e1e93d2530ac820981c93986d64e46faa3